WORKSPACE

```diff
@@ -454,12 +454,6 @@ go_repository(
     tag = "v4.2.1",
 )
 
-go_repository(
```

Comment:
go mod tidy
then
bazel run //:gazelle -- update-repos --from_file=go.mod
?

Reply:
I did tidy but not gazelle, gazelle seems bumping versions of some dependencies automatically (in addition to just add/remove dependencies). I updated with gazelle produced diff to check if the version change will cause incompatibility issue. Otherwise, we can keep the gazelle produced diff.